src/pybind/mgr/dashboard/controllers/rgw.py

```diff
@@ -1295,3 +1295,30 @@ def get_user_list(self, zoneName=None):
         multisite_instance = RgwMultisite()
         result = multisite_instance.get_user_list(zoneName)
         return result
+
+@APIRouter('/rgw/Topics', Scope.RGW)
```

Comment:
```suggestion
@APIRouter('/rgw/topic', Scope.RGW)
```
All smallcase  for api routes and not plural, sorry I added that by mistake in prev comment

Reply:
done